packages/tpetra/core/inout/mmio_Tpetra.c

```diff
@@ -0,0 +1,382 @@
+/* 
```

Comment:
@seheracer I couldn't find any info on what license governs this software.  Any idea what it is?